### DISCUSSION TITLE
如何添加对mongodb的支持

### DISCUSSION THREAD

**Discussion by Jv-Juven** [zhangyuang/ssr#185] [Q&A]
找了一下文档，没有发现对数据库的支持说明，试了一下eggjs中添加egg-mongoose插件的做法，貌似并不管用。请指导一下~谢谢

**Comment by zhangyuang** [comment]
跟ssr无关，自行查看midway/egg相关文档。（我理解不需要要用 `egg-mongoose` 直接用原始的 `mongoose` 就行了，也可以尝试用nest-js
